O/OptFrame/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name    = "OptFrame"
+version = v"6.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource(
+        "https://github.com/OptFrame/OptFrame.git",
+        "3b57dcd24bc4dff2abf0ef28010bf736509eefa5",
+    )
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/OptFrame/
+
+make optframe_lib
+
+install -Dvm755 build/optframe_lib.so "${libdir}/optframe_lib.${dlext}"
```

Comment:
It seems this only installs this one file?

But that means it does not install a license file, which is a requirement...

Otherwise, the tests here seem to have passed last time, and we just missed this PR was ready :-(. Please don't hesitate to ping someone next time this happens.